lib/pry/prompt.rb

```diff
@@ -94,11 +94,10 @@ def prompt_name(name)
       end
     end
 
-    add(:default, <<DESC) do |context, nesting, _pry_, sep|
-The default Pry prompt. Includes information about the current expression
-number, evaluation context, and nesting level, plus a reminder that you're
-using Pry.
-DESC
+    add 'default',
```

Comment:
Is there anything wrong with this approach?

```rb
add(
  :default,
  <<DESC
The default Pry prompt. Includes information about the current expression
number, evaluation context, and nesting level, plus a reminder that you're using Pry
DESC,
  %[> *]
) do |...|
...
end
```

The benefits I see: 
* wrapping manually is trivial since line length corresponds to the actual line length and can be shown by your text editor
* As result, no need to add `word_wrap`, which is another public method we have to support

Reply:
Alternatively,

```rb
add(
  :default,
  "The default Pry prompt. Includes information about the current expression" \
  "number, evaluation context, and nesting level, plus a reminder that you're using Pry",
  %[> *]
) do |...|
...
end
```

Simple change without shooting sparrows with cannons.

Reply:
first one has same problem, second one looks good, i'll update

Reply:
oh, i missed your comment about `word_wrap`, i disagree there since custom prompts won't necessarily follow the 80 col rule. the 2nd example doesn't solve the 80 col problem too.

i find `word_wrap` to be a useful addition outside this case too, i can imagine a plugin author/developer wanting to wrap the output of their commands.

anyway let me know what you want to do, before i make any more changes.

Reply:
> wrapping manually is trivial since line length corresponds to the actual line length and can be shown by your text editor

what makes you think an author of a prompt will follow the 80 col rule?
unless they use the heredoc syntax you were using, it's not actually so easy and trivial, very easy to look and be unaware of that. `word_wrap` solves it for all prompts, ours and others, in like 6 lines of code. it is generally useful method too imo, especially for terminals

Reply:
This discussion is getting a little out of scope this PR. First you fix some highlighting and indentation, now we talk about `word_wrap`. Let's just fix what needs to be fixed, and move on and revisit this when needed.

Reply:
well, the PR evolved so that `word_wrap` made sense, and it is a small PR, so i think it is worth considering now. i have already explained why i believe it is a good idea.

Reply:
the main reason i believe it is a good idea is that it will ensure all prompts are printed within 80 cols and guarantees pry prompts and custom prompts alike will be readable and consistent, as well as making it easier to define prompts without using heredoc strings to get the same desired formatting.

Reply:
Please, if you find it useful, let's add it separately and use it where it's relevant. Let's keep this PR simple.
Besides:

* `word_wrap` does not handle colour sequences which Pry heavily uses (even with Prompt)
* creating a global helper to be used only at one place in our private part of the code is rather wasteful. If you really want it, create a separate PR and identify more places where it's useful. People don't write prompts every day and it's okay not to have this helper for something that's people write maybe once a year

Reply:
sure i'll work on the the colour sequence issue.

`word_wrap` is meant to be a command helper and `Text` seemed like the best module to put it, `Text` extending itself makes it a 'global helper' sure but all Text command helpers are that way. this module seems like the most appropriate place to put command helpers that deal with strings/text.

Reply:
Let’s merge this without auto wrapping and then I’d be happy to review the word wrap PR 🙏

Reply:
okay, squashed and pushed with just the fix for syntax highlighting.